martin/src/pmtiles/mod.rs

```diff
@@ -361,3 +390,14 @@ impl PmtFileSource {
         Self::new_int(id, path, reader).await
     }
 }
+
+/// Interpret an environment variable as a [`bool`]
+///
+/// This ignores casing, but interprets bad utf8 encoding as `false`.
+fn get_env_as_bool(key: &'static str) -> bool {
+    std::env::var_os(key)
+        .unwrap_or_default()
+        .to_ascii_lowercase()
+        .to_str()
+        .is_some_and(|val| matches!(val, "1" | "true"))
```

Comment:
this feels a bit less readable than the usual `val == "1" || val == "true"`...  not a biggie though, just curious why use matches for string comp?

Reply:
Not a real reason. Was just the first thing that came to mind when I noticed that "wow, this code looks hideous".

```suggestion
        .is_some_and(|val| val == "1" || val == "true")
```